internal/provider/models/destinations/file_consolidation.go

```diff
@@ -0,0 +1,61 @@
+package destinations
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework-validators/int64validator"
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/int64default"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
```

Comment:
Please use `basetypes` (`basetypes.BoolValue`, etc).  `types` is just a wrapper for basetypes and the names have gotten confusing in the past.  The TF framework is hard enough to understand without this indirection.

Seeing something like `basetypes.BoolValue` in the type can help match up the back-and-forth conversion when pulling out values with a type with the corresponding `ValueBool` type.

Reply:
Wow, thats confusing! Fixed it :)